src/sanitizer.js

```diff
@@ -186,9 +190,19 @@ export function sanitizeHtml(html) {
         }
         return;
       }
+      const bindAttribsIndices = map();
```

Comment:
`hasOwn` isn't necessary if we use a map, just `bindAttribsIndices[i]` will be sufficient to test.